src/utils/user-helpers.ts

```diff
@@ -40,6 +41,15 @@ export const getOrcidUri = (identifiers: CristinPersonIdentifier[] = []) => {
   return orcid ? `${ORCID_BASE_URL}/${orcid}` : '';
 };
 
+export const getVerificationStatus = (verifiedStatus: boolean | undefined) => {
+  if (verifiedStatus) {
+    return VerificationStatus.Verified;
+  } else if (!verifiedStatus) {
```

Comment:
Må ha en eksplisitt sjekk mot `false` her, siden både `false` og `undefined` vil bli `true` når man bruker `!`
```suggestion
  } else if (verifiedStatus === false) {
```